packages/paste-core/components/modal/src/Modal.tsx

```diff
@@ -72,19 +71,13 @@ export interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
   size: Sizes;
   initialFocusRef?: React.RefObject<any>;
   ariaLabelledby: string;
-  // Adds an 80px margin to the root elements on the Console application
-  __console_patch?: boolean;
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const getAnimationStates = (isConsole: boolean): any => ({
+const getAnimationStates = (): any => ({
```

Comment:
sugestion: this doesn't need to be a function anymore

Reply:
I refactored the function to be an object but I'm getting this ts error
<img width="1038" alt="image" src="https://user-images.githubusercontent.com/75342690/193123022-d09d5841-4fe1-434a-b00a-9bffffce43f0.png">

Reply:
Ok let's leave it as a function